### PR TITLE
Fix rejected behavior, add verbose flag, add zip alias

### PR DESCRIPTION
**Pull request by louisgv**
Ref: #37

### CODE REVIEW

File: keys.schema.json [hunk 1]
```diff
@@ -65,7 +65,7 @@
         },
         "extId": {
           "type": "string",
-          "description": "Get it from https://addons.mozilla.org/addon/EXT_ID. If it is embedded in your manifest under gecko.id, omit this property."
+          "description": "This is the extension UUID, get it from https://addons.mozilla.org/addon/{ext-name}/edit, under Technical Details. If it is embedded in your manifest under gecko.id, omit this property."
```
Comment (ColdSauce) [inline keys.schema.json:68]: Same as your other PR, I feel like we should change this name as it's confusing. Something like `extUuid` might be less confusing.

File: keys.template.json [hunk 2]
```diff
@@ -6,12 +6,12 @@
...
   },
   "firefox": {
-    "extId": "123",
+    "extId": "aaaaaaa-aaaa-bbbb-cccc-dddddddddddd",
     "apiKey": "abcd",
     "apiSecret": "efgh"
   },
   "edge": {
-    "extId": "123",
+    "extId": "aaaaaaa-aaaa-bbbb-cccc-dddddddddddd",
```
Comment (ColdSauce) [inline keys.template.json:14]: should potentially rename this to `productId`

File: src/main.ts [hunk 3]
```diff
@@ -26,9 +26,16 @@ async function run(): Promise<void> {
...
     // Path to the zip file to be deployed
-    const artifact = getInput("artifact")
+    const artifact = getInput("zip") || getInput("artifact")
     const versionFile = getInput("version-file")
 
+    const verbose = !!getInput("verbose")
+
+    if (verbose) {
+      // All internal logging goes to info
+      console.log = info
```
Comment (ColdSauce) [inline src/main.ts:36]: I don't think this is necessary.

Looking at the info function's [code](https://github.com/actions/toolkit/blob/27f76dfe1afb2b7e5e679cd8e97192d34d8320e6/packages/core/src/core.ts#L290) it's merely the following: 

```
export function info(message: string): void {
  process.stdout.write(message + os.EOL)
}
```

`info` is a wrapper that writes strings to stdout, like `console.log`, but doesn't offer the other nice properties that `console.log` has (`console.log("some string", someObj)` etc) and monkey patching might be confusing from a maintanence perspective

I suggest we just don't do this

File: tests/main.test.ts [hunk 4]
```diff
@@ -18,8 +18,16 @@ test("happy path", async () => {
   }
 
   const output = execFileSync(execPath, [ip], options).toString("utf-8")
-
   expect(output).toMatchSnapshot()
+
+  // try {
```
Comment (ColdSauce) [inline tests/main.test.ts:23]: Just flagging this. Not sure if you wanted it here or not.

File: tests/main.test.ts [hunk 5]
```diff
@@ -34,3 +42,16 @@ test("will fail if no zip found", async () => {
...
+test("get verbose flag successfully", async () => {
+  env["INPUT_KEYS"] = `{"chrome": {}}`
+  env["INPUT_ZIP"] = "artifacts.zip"
+  env["INPUT_VERBOSE"] = "true"
+
+  const options: ExecFileSyncOptions = {
+    env
+  }
+
+  const output = execFileSync(execPath, [ip], options).toString("utf-8")
```
Comment (ColdSauce) [inline tests/main.test.ts:55]: Curious - why is this called `ip`? Initial point?

File: tests/main.test.ts [hunk 6]
```diff
@@ -34,3 +42,16 @@ test("will fail if no zip found", async () => {
...
+  expect(output).toMatchSnapshot()
```
Comment (ColdSauce) [inline tests/main.test.ts:56]: That's cool, never seen snapshots before, but they're awesome

**Review by ColdSauce — CHANGES_REQUESTED** [review]

File: tests/main.test.ts (hunk 5 above)
Comment (louisgv) [reply tests/main.test.ts:55]: Renamed it to indexScript

File: keys.schema.json (hunk 1 above)
Comment (louisgv) [reply keys.schema.json:68]: Let's consider in v2

**Pull request merged**